components/unzipped/dashboard/Kanban/ProjectusersDropdown.js

```diff
@@ -0,0 +1,230 @@
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import Typography from '@mui/material/Typography';
+import styled from 'styled-components';
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import BadgeOutlinedIcon from '@mui/icons-material/BadgeOutlined';
+
+const StyledMenu = styled((props) => (
+    <Menu
+        elevation={0}
+        anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'right',
+        }}
+        transformOrigin={{
+            vertical: 'top',
+            horizontal: 'right',
+        }}
+        {...props}
+    />
+))(({ theme }) => ({
+
+    '& .MuiPaper-root': {
+        borderRadius: 6,
+        marginTop: "8px",
+        minWidth: 400,
+        maxWidth: 400,
+        maxHeight: 250,
+        color: "#737373",
+        boxShadow:
+            'rgb(255, 255, 255) 0px 0px 0px 0px, rgba(0, 0, 0, 0.05) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px',
+        '& .MuiMenu-list': {
+            padding: '2px 0',
+        },
+    },
+}));
+
+
+const USERS_ARR = [
+    {
+        email: "jasonmaynoard@gmail.com",
+        firstName: "Jason",
```

Comment:
Again, the users here should be the owner of the project and any employees that have an open contract with the project. This way any employees can be assigned to tickets, or the ticket can be unassigned, or the project owner can assign themselves to the ticket.